.github/workflows/ci.yaml

```diff
@@ -11,6 +11,23 @@ on:
       - created
 jobs:
 
+  hlint:
```

Comment:
I moved these up to the top because they finish much quicker so the jobs are roughly in time order.